README.md

```diff
@@ -190,4 +191,13 @@ If the user logs out of your application, you should unset the user id:
 
 ```java
 Sift.unsetUserId();
-```
+``` 
+
+<a name="submitting_an_issue"></a>
+## Submitting a PR or Issue
+
+If you find issues or want to improve the SDK please create a issue or PR. We ask you provide the following in your submission.  
+
+- Provide Context: Explain how you discovered this issue and code samples.
```

Comment:
nit: I am proposing a small change to hint developers to include steps:
`Explain how you discovered this issue(include steps if possible) and code samples.`